girder/models/model_base.py

```diff
@@ -806,9 +937,17 @@ def setGroupAccess(self, doc, group, level, save=False):
                      Set this to False if you want to wait to save the
                      document for performance reasons.
         :type save: bool
+        :param flags: List of access flags to grant to the group.
+        :type flags: specific flag identifier, or a list/tuple/set of them
+        :param currentUser: The user performing this action. Only required if attempting
+            to set admin-only flags on the resource.
+        :type currentUser: dict or None
         :returns: The updated resource document.
+        :param force: Set this to True to set the flags regardless of the passed in
+            user's permissions (only matters if flags are passed).
```

Comment:
I think it'll be clearer to change the docstring `user's` => `currentUser's`.  More for consistency with a parallel change below, where it matters more.